docs/upgrading/v3.mdx

```diff
@@ -12,75 +16,101 @@ Bref 3.0 now requires PHP 8.2 or greater.
 
 ### Composer Dependencies
 
-You should update the `bref/bref` dependency in your application's composer.json file:
+Update all Bref packages in your `composer.json` file from `^2` to `^3`, for example:
 
-```diff
--    "bref/bref": "^2",
-+    "bref/bref": "^3",
+```json filename="composer.json"
+"require": {
+    "bref/bref": "^3",
+    "bref/laravel-bridge": "^3",         // if you use Laravel
+    "bref/symfony-bridge": "^3",         // if you use Symfony
+    "bref/extra-php-extensions": "^3"    // if you use extra extensions
```

Comment:
This doesn't seem accurate, given that there isn't even a `^2` version of bref/extra-php-extensions.